### DISCUSSION TITLE
Mac Os Sonoma fitting version of vs codium

### DISCUSSION THREAD

**Discussion by miimiiaa** [VSCodium/vscodium#1783] [Q&A]
So I am a newbie with VS Codium and I have a new laptop.
It is a macbook so it runs on sonoma 14.1 at the moment. I just installed VSCodium.x64.1.85.1.23348 without being sure if I am downloading the right version. Now that I opened the program, it told me to download a newer version and if I go on that link it brings me to visual studio code. But as far as I know this is the official version that is not fully open source and so on. 
All the information I could find did not perfectly fit to my problem/question...
I just try to make sure, that I use the right version.

So my question is: Which version is the best to work with on my computer? should I download the vs code (VSCode-darwin-universal) from the link or should I keep the version I already have or should I download VSCodium.arm64.1.85.1.23348?

Thank you in regards for any help!!!

**Comment by Jeong-ilKajokaya** [comment]
Hi! Here’s some guidance for using VSCodium on your MacBook (Sonoma 14.1).

---

### 1. Choosing the Right Version

VSCodium releases two main Mac builds:

* **x64 build**: For Intel-based Macs.
* **arm64 build**: For Apple Silicon (M1/M2) Macs.

Since most modern MacBooks are Apple Silicon:

* **If your Mac is Apple Silicon (M1/M2)**: Use `VSCodium.arm64`. This is optimized for your hardware and runs faster.
* **If your Mac is Intel-based**: Stick with `VSCodium.x64`.

You can check your Mac architecture in ` -> About This Mac -> System Report -> Hardware Overview -> Processor Name`.

---

### 2. Updating VSCodium

* VSCodium will sometimes suggest updates pointing to the official VSCode website. **Do not download VSCode** if you want the fully open-source build.
* Instead, download the latest **VSCodium release** for your architecture from:
  [https://github.com/VSCodium/vscodium/releases](https://github.com/VSCodium/vscodium/releases)

---

### 3. Summary Recommendations

* Keep using **VSCodium**, not VSCode, if you want fully open-source.
* Use **VSCodium.arm64** for Apple Silicon Macs.
* Always get updates from the official **VSCodium GitHub releases**, not the VSCode site.
* x64 build is fine for Intel Macs, but arm64 is preferred on Apple Silicon.

This will ensure you have a fully open-source, optimized version for your MacBook.
